apps/win32/metro.config.js

```diff
@@ -5,12 +5,15 @@
  * @format
  */
 
-const { defaultWatchFolders, exclusionList } = require('@rnx-kit/metro-config');
+const { defaultWatchFolders, exclusionList, resolveUniqueModule } = require('@rnx-kit/metro-config');
 const { getDefaultConfig } = require('metro-config');
 
+const [reactIs, reactIsExcludePattern] = resolveUniqueModule('react-is');
+
 const blockList = exclusionList([
   // Exclude build output directory
   /.*\/apps\/win32\/dist\/.*/,
+  reactIsExcludePattern,
```

Comment:
If I may ask, why do we need to exclude this on win32 only?

Reply:
Win32 was the only app that didn't. It failed to start because `rnx-start` is reporting it now 😄